marklogic-data-hub/src/main/java/com/marklogic/hub/impl/ArtifactManagerImpl.java

```diff
@@ -29,18 +30,20 @@
 import java.nio.file.Path;
 import java.nio.file.Paths;
 import java.util.Iterator;
+import java.util.LinkedList;
+import java.util.List;
 
 @Component
```

Comment:
I think "Component" can be removed, right? I'd rather see the tests that are autowiring this in just instantiate an instance when it needs it (there are only 3 of them).